merlin/dag/base_operator.py

```diff
@@ -46,12 +46,14 @@ def compute_selector(
         self,
         input_schema: Schema,
         selector: ColumnSelector,
-        parents_selector: ColumnSelector,
-        dependencies_selector: ColumnSelector,
+        parents_selector: ColumnSelector = None,
+        dependencies_selector: ColumnSelector = None,
     ) -> ColumnSelector:
+        selector = selector or ColumnSelector("*")
```

Comment:
this line suggests that `selector` might also be None, which isn't what the typehint above would suggest. Is the intention to make `selector` also optional?

Reply:
In that case would it make sense to update the typehint in the args from `ColumnSelector` to `Optional[ColumnSelector]`?